src/main/scala/org/constellation/primitives/ChannelMessage.scala

```diff
@@ -34,10 +32,18 @@ case class ChannelMessageMetadata(
   snapshotHash: Option[String] = None
 )
 
+case class ChannelMetadata(
+  channelOpen: ChannelOpen,
+  genesisMessageMetadata: ChannelMessageMetadata,
+  totalNumMessages: Long = 0L
+)
+
 case class ChannelMessage(signedMessageData: SignedData[ChannelMessageData])
 
 object ChannelMessage {
 
+  val logger = Logger("ChannelMessage")
```

Comment:
unless you want a custom name, I think having the object or class extend StrictLogging is a nicer pattern. It also names the logger with the package name in front which is useful for tweaking log levels, because you can say org.constellation.* is debug, etc.

Reply:
Fixed